src/Libraries/PythonNodeModels/PythonEngineSelector .cs

```diff
@@ -9,6 +9,7 @@ namespace PythonNodeModels
     /// </summary>
     public enum PythonEngineVersion
     {
+        Unspecified,
```

Comment:
this should go at the end of the list as this change re-orders all the existing enums - ie:
Ironpython2 used to = `0`, but after this change it is `1`.

Reply:
Technically this only breaks binary compatibility. I put this in the balance with following best practices, and opted for the latter. Let me know if you'd rather change this anyway.

Reply:
my experience with this type of change is pretty complex confusion - code which was previously compiled against Dynamo 2.7 and selected engine ironPython2 will now be selecting unspecified.

I guess it's unlikely that any clients besides our own would ever be in this situation though.

If you've already thought about it then I'm fine with your decision. 👍